src/plugins/kernel_install/utils.sh

```diff
@@ -550,32 +550,85 @@ function do_uninstall()
   fi
 }
 
+# Element existence check
+# Checks if element belongs to given array
+#
+# @target_element element being checked
+# @array target array
+# returns 1 if element is in array and 0 otherwise
```

Comment:
Add a blank line between the arguments and the return documentation. Also, make the returns section like the below:

```bash
# Return:
# Returns X in case Y....
```